cmd/observer/node_builder/observer_builder.go

```diff
@@ -938,6 +1333,33 @@ func (builder *ObserverServiceBuilder) enqueueRPCServer() {
 		builder.rpcConf.TransportCredentials = credentials.NewTLS(tlsConfig)
 		return nil
 	})
+	builder.Module("creating grpc servers", func(node *cmd.NodeConfig) error {
+		builder.secureGrpcServer = grpcserver.NewGrpcServerBuilder(node.Logger,
+			builder.rpcConf.SecureGRPCListenAddr,
+			builder.rpcConf.MaxMsgSize,
+			builder.rpcMetricsEnabled,
+			builder.apiRatelimits,
+			builder.apiBurstlimits,
+			grpcserver.WithTransportCredentials(builder.rpcConf.TransportCredentials)).Build()
+
+		builder.unsecureGrpcServer = grpcserver.NewGrpcServerBuilder(node.Logger,
+			builder.rpcConf.UnsecureGRPCListenAddr,
+			builder.rpcConf.MaxMsgSize,
+			builder.rpcMetricsEnabled,
+			builder.apiRatelimits,
+			builder.apiBurstlimits).Build()
+
+		return nil
+	})
+
+	builder.Module("async register store", func(node *cmd.NodeConfig) error {
```

Comment:
is this needed for this PR? if not, let's leave it out until we add script execution